transform/internal/jsonpatch/compare.go

```diff
@@ -35,6 +37,19 @@ func EqualOperation(operation1, operation2 jsonpatch.Operation) bool {
 		if path1 != path2 {
 			return false
 		}
+		if (operation1.Kind() == "move" || operation1.Kind() == "copy") {
```

Comment:
I wonder if we are going to expect this function to ballon, if we create a method for each check (path, from, value... etc. and then use a switch on type to pass the operations for the checks that need to be made for that given kind? 

I think that maybe as a follow-up but I wonder if this will allow us to more easily add and see what is going on here in regards to jsonpatch spec?

Either I see it as follow up wondering if you think an issue makes sense?

Reply:
At the moment, we only have one example of this -- the "from" field is only relevant for "move" and "copy". If `Kind()` is "move" or "copy", then from must be equal as well. Another way of handling this would be to ignore kind and just mandate that "from" fields must either be equal or both empty. Then we could ignore Kind in the check.